flexmeasures/data/models/planning/storage.py

```diff
@@ -246,13 +246,16 @@ def compute(self, skip_validation: bool = False) -> pd.Series | None:
             commitment_upwards_deviation_price,
             initial_stock=soc_at_start * (timedelta(hours=1) / resolution),
         )
+
         if scheduler_results.solver.termination_condition == "infeasible":
             # Fallback policy if the problem was unsolvable
```

Comment:
Here we should log a one-line warning that at least let's the host see *who* (current_user) and *what* (sensor_id) was affected. The *why* can't be answered right now.

Reply:
Good idea! that way we can notice in the logs